### DISCUSSION TITLE
VRM 1.0対応表記について

### DISCUSSION THREAD

**Discussion by RearAlice** [pixiv/vroid-sdk-developers#81] [Q&A] [answered]
先日ユーザーより質問をいただいたので共有させていただきます。

### 質問
当方でも先日SDK 2.0でVRM1.0に対応しました。
しかし、ユーザーより頂いたものとして、VRoidHub側でモデルのページでは以下の表記が残っています。
![タイトルなし](https://github.com/pixiv/vroid-sdk-developers/assets/120567699/46707a2a-51ec-4399-a705-dcdd64fc647a)
これは、VRoidHub側のシステムがまだ対応していないからなのでしょうか？

### 要望
アプリページの下にある文言ですが
![タイトルなし1](https://github.com/pixiv/vroid-sdk-developers/assets/120567699/3853513c-94f6-4a98-b940-a221f1a140ab)
「※VRM1.0でVRoid Hubにアップロードしたモデルはまだ利用できません」という文言を[アプリケーション管理ページ](https://hub.vroid.com/oauth/applications)でオプションで付けたり消したりできるようにできたらいいなぁと思います。

**Comment by ytomi-pixiv** [comment, marked as answer]
ご指摘ありがとうございます。
不便をおかけしてしまい大変申し訳ありません。

添付画像のようにVRoid Hub内の表現を変更しました。

①モデル詳細ページ（VRM1.0モデル）
> VRM1.0のモデルを連携アプリケーションで利用する場合、アプリケーションがVRM1.0に対応しているかご確認ください。

②アプリ詳細ページ
> VRM1.0のモデルを利用できるかどうかは、アプリケーションがVRM1.0に対応しているかご確認ください。

アプリ詳細ページでのVRM1.0対応有無の表示については、近日中に、アプリケーションで利用しているSDKのバージョンからVRM1.0に対応しているかどうかを判定し、アプリ詳細ページに表示する方針で考えております。
いましばらくお待ちいただけますと幸いです。

![image (66)](https://github.com/pixiv/vroid-sdk-developers/assets/92707708/48ad88dd-4291-4bfd-a2a8-75935bac085e)
![image (67)](https://github.com/pixiv/vroid-sdk-developers/assets/92707708/2e398d8d-f2dd-4507-b6a9-8d8f0630dc70)

ご連絡ありがとうございました。大変助かります。
上記変更に関しても、不明点などありましたらお知らせください。

どうぞ引き続き、よろしくお願いいたします。

**Reply by RearAlice** [reply]
早速のご対応ありがとうございます。
しかし、SDKで判断するとSDK最新だけどアプリ内では未対応とか、VRM1.0モデル読み込むとエラー出るので一旦未対応ですっていう表示に戻したい時等の場合の例外が発生する可能性があるので、当初の要望のように選べればいいかもしれません。（一応SDK判断でデフォルトOFFとかでいいかもしれませんが。）

もしくは、利用できるモデルについての場所に対応VRMバージョンという項目増やして「VRM0.x ,1.0対応」とか表示だせるといいかもしれません。

**Reply by ytomi-pixiv** [reply]
お返事いただいてから時間が空いてしましましたが、以下のアナウンスのように、アプリケーション設定で選べるようにアップデートする予定です。
フィードバックいただきありがとうございました。大変助かります。

不明点等ありましたらお知らせください。どうぞ引き続きよろしくお願いいたします。
https://github.com/pixiv/vroid-sdk-developers/discussions/92
